src/coprocessor/endpoint.rs

```diff
@@ -252,13 +133,10 @@ impl Host {
         let mut tracker = t.tracker;
         let ranges = req.take_ranges().into_vec();
 
-        let pool_and_ctx_pool = match req.get_context().get_priority() {
-            CommandPri::Low => &mut self.low_priority_pool,
-            CommandPri::High => &mut self.high_priority_pool,
-            CommandPri::Normal => &mut self.pool,
-        };
-        let pool = &pool_and_ctx_pool.pool;
-        tracker.ctx_pool(pool_and_ctx_pool.contexts.clone());
+        let priority = readpool::Priority::from(req.get_context().get_priority());
+        let pool = self.pool.get_pool_by_priority(priority);
+        let ctxd = pool.get_context_delegators();
+        tracker.ctx_pool(&ctxd);
```

Comment:
It seems we clone `context_delegators` twice on L138 and L139, why not move `ctxd` directly instead of `clone` on L139?

Reply:
Ok